### PR TITLE
DROOLS-3141: [DMN Designer] Decision Table: Add additional Properties to drill-down

### PR DESCRIPTION
**Pull request by manstis**
Depends on https://github.com/kiegroup/kie-wb-common/pull/2196 to be merged.

See https://issues.jboss.org/browse/DROOLS-3141

This also incorporates https://issues.jboss.org/browse/DROOLS-3118

### CODE REVIEW

**Comment by manstis** [comment]
@jomarko @karreiro This PR is good for review now :-)

**Comment by manstis** [comment]
@karreiro I don't see what code could be shared/reused? 

The 2 methods take different parameters (that do not extend a common base class).

**Comment by karreiro** [comment]
@manstis Yes, they don't extend a common base class.. but I think that they could implement something like this:
```java
interface DMNUnaryTest {

    Id getId();

    Text getText();

    Description getDescription();

    ExpressionLanguage getExpressionLanguage();
}
```
Am I missing something?

Thanks.

File: kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/resources/org/kie/workbench/common/dmn/api/resources/i18n/DMNAPIConstants.properties [hunk 1]
```diff
@@ -26,15 +26,25 @@ org.kie.workbench.common.dmn.api.definition.v1_1.Decision.title=DMN Decision
...
+org.kie.workbench.common.dmn.api.definition.v1_1.InputClause.title=DMN Input Clause
+org.kie.workbench.common.dmn.api.definition.v1_1.InputClause.description=DMN Input Clause
+org.kie.workbench.common.dmn.api.definition.v1_1.InputClause.inputValues=Constraint
+org.kie.workbench.common.dmn.api.definition.v1_1.InputClauseUnaryTests.text=Constraint value
+org.kie.workbench.common.dmn.api.definition.v1_1.OutputClause.title=DMN Output Clause
+org.kie.workbench.common.dmn.api.definition.v1_1.OutputClause.description=DMN Output Clause
+org.kie.workbench.common.dmn.api.definition.v1_1.OutputClause.outputValues=Output values
+org.kie.workbench.common.dmn.api.definition.v1_1.OutputClause.defaultOutputEntry=Default output
+org.kie.workbench.common.dmn.api.definition.v1_1.OutputClauseUnaryTests.text=Output value
+org.kie.workbench.common.dmn.api.definition.v1_1.OutputClauseLiteralExpression.text=Default output value
```
[the remark this answers is not in the document]
Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/resources/org/kie/workbench/common/dmn/api/resources/i18n/DMNAPIConstants.properties]: @jomarko Hi, thanks for the pre-review :-)

> If multiple cells selected..

See https://github.com/kiegroup/kie-wb-common/pull/2200#pullrequestreview-167903793 (I've already said about multiple-cell selection.. probably a new JIRA?)

> Does new stunner forms support hints?

Yes it does, but we've not added any yet... a new JIRA?

**Comment by manstis** [comment]
@karreiro New commit added with `IsUnaryTests` and `IsLiteralExpression` to support maximum re-use of the _property converters_ on the backend (you decide whether the value added was worth the effort!).

**Comment by jomarko** [comment]
Filed issues according to [comment](https://github.com/kiegroup/kie-wb-common/pull/2201#discussion_r228479158)
- [DROOLS-3199](https://issues.jboss.org/browse/DROOLS-3199)
- [DROOLS-3200](https://issues.jboss.org/browse/DROOLS-3200)

Manual re-check pending.

**Comment by manstis** [comment]
Hi @jomarko thank-you for your candid feedback.

To be honest IDK what values should be entered in what fields either. However I do know that the apparent non-saving of some fields is caused by our old (and really, really, really problematic) https://issues.jboss.org/browse/JBPM-7166. The _new_ properties are saved but on the wrong object :-(

Add a value for **all** properties and select a different node/save/close/re-open or whatever and you'll see some values are saved but not for the correct object (e.g. add values for "Description", "Output value" and "Expression Language" on **all** properties of an {{OutputClause}} column and you'll see when clicking a different node etc that only some were actually set :-(

Furthermore I'd like to propose all runtime issues are now reported on EPIC DROOLS-3203. I've added a few individual JIRA that already exist but I think it's important now (so close to 7.2) to track these issues as our priority. Could you please add a new JIRA there for the errors you report here?

My idea is to have @etirelli or @tarilabs review the issues and advise what appears to be broken in the XML we're creating.. it worked well so far and I hope works as efficiently as possible for 7.2....

**Pull request merged**